_spinnaker/halyard_secrets.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Revision Control Configurations
+order: 25
+---
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+## Revision Control using Open Source Halyard
+OSS Spinnaker and Halyard stores secrets prompted by **hal commands** into `~/.hal/config.yml` and other `~/.hal/` folders/files. 
+
+**Our recommendation** is that you take the `~/.hal/` directory and treat this like a secret, stored in your secret store of choice. 
+
+Another possibility is to store `~/.hal/` into your revision control, however we do not recommend this approach as, you'll be storing secrets in your revision control.
+
+Another method is to turn your `~/.hal/config.yml` and other files into a template and to build a process around hydrating/baking it, then doing a `hal deploy apply`.
+> Warning: you may lose the ability to run `hal` commands, because halyard may not understand your templates.
+
+
+## Revision Control using Armory Halyard
+With Armory Halyard, we've done the hard part for you by splitting up configs and secrets.
+
+We do this by having:
```

Comment:
Is this currently implemented? How does it work? Does Armory Halyard handle this _for you_ or are you expected to drop secrets into these files/directories and then reference them?

Reply:
It's not implemented (https://armory.atlassian.net/browse/ENG-2701) and probably 2 sprints away.

Reply:
In that case we should probably remove it until it's ready.

Reply:
We may want to describe how to manually manage this aspect -- and then swap it out for an overview of our automated process once we have it in place.

Reply:
There's no automated way of doing this currently. We'll just need to remove this section then.